bigtop-bigpetstore/bigpetstore-transaction-queue/README.md

```diff
@@ -87,8 +87,7 @@ OR Replace the file path with a REST API root (it will jsonify the transactions,
 From docker
 
 ```
-docker run -t -i jayunit100/bigpetstore-load-generator /opt/bigpetstore-transaction-queue-1.0/bin/bigpetstore-transaction-queue
- http://localhost:3000/restapi/rpush/ 1 5 10000 123
+docker run -t -i jayunit100/bigpetstore-load-generator http://localhost:3000/restapi/rpush/ 1 5 10000 123
```

Comment:
I'm wondering whether can be switch to bigtop repository instead of jayunit100. But anyway we can deal with this in separated JIRA.